README.rst

```diff
@@ -12,10 +12,20 @@ at https://github.com/enovance/edeploy.
 
 eDeploy-roles is a set of tools to build roles to be used with eDeploy.
 
+Requirement
+-----------
+
+You need to get the ``edeploy`` repository and generate the ``base`` role first::
+
+  $ git clone http://github.com/enovance/edploy
+  $ sudo ln -s $PWD/edeploy /srv/edeploy
```

Comment:
Yes please. Installing eDeploy in /srv/edeploy is the way to go, for dev and for production.